### PR TITLE
Adds CIFS volume plugin for coreos nodes if kubelet_enable_cifs_mount variable is set.

### PR DESCRIPTION
**Pull request by Ishashanim**

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -40,3 +40,15 @@
...
   service: name=kubelet.service state=started enabled=yes
+
+- name: make fstab directory for cifs plugin
+  become: yes
+  file:
+    path: "{{kubelet_volume_plugin_dir}}/fstab~cifs"
+    state: directory
+  when: kubelet_enable_cifs_mount
+
+- name: install cifs plugin
```
Comment (puneetkatyal) [inline tasks/main.yml:51]: Does Kubelet not need to be restarted for this? If it does, then these tasks should call the appropriate handler.

Comment (Ishashanim) [reply tasks/main.yml:51]: Puneet, we don't need to restart Kubelet for this. The presence of this file on the nodes itself, will allow pods to mount CIFS volume.

File: tasks/main.yml [hunk 2]
```diff
@@ -40,3 +40,15 @@
 - name: Start kubelet
   become: yes
   service: name=kubelet.service state=started enabled=yes
+
+- name: make fstab directory for cifs plugin
+  become: yes
+  file:
+    path: "{{kubelet_volume_plugin_dir}}/fstab~cifs"
+    state: directory
+  when: kubelet_enable_cifs_mount
```
Comment (puneetkatyal) [inline tasks/main.yml:49]: I think you need to apply the Ansible `bool` filter to this check else `False` or `True` is treated as string making this always true.
Sorry for not noticing this earlier.

Comment (Ishashanim) [reply tasks/main.yml:49]: Puneet, If we use False for this variable, it will skip installing the plugin.

**Pull request merged**